params/config.go

```diff
@@ -553,6 +583,10 @@ type ChainConfig struct {
 	SocialBlock        *big.Int `json:"socialBlock,omitempty"`        // Ethereum Social Reward block
 	EthersocialBlock   *big.Int `json:"ethersocialBlock,omitempty"`   // Ethersocial Reward block
 
+	MCIP0Block *big.Int `json:"mcip0Block,omitempty"` // Musicoin default block; no MCIP, just denotes chain pref
```

Comment:
Using `MCIP0-` instead of `MusicoinBlock-`. This breaks pattern with, eg. `SocialBlock`, but keeps with `EIPXX-`.